oak_restricted_kernel/src/payload.rs

```diff
@@ -54,6 +55,7 @@ struct Binary {
 /// Representation of an Restricted Application that the Restricted Kernel can run.
 pub struct Application {
     binary: Binary,
+    hash: [u8; 32],
```

Comment:
we have been using the term `digest` to refer to the output of the hash function.